### DISCUSSION TITLE
Who knows coding? Please make a plugin for Krita or Photoshop!

### DISCUSSION THREAD

**Discussion by nestoris** [nestoris/Win98SE#19] [Ideas]
I need to have a Krita or Photoshop plugin for previewing an image in 256-color palette with gradients as diffusions or patterns (Floyd Steinberg, even/odd, etc). It may be as a custom layer or menu item "8-bit preview". So, I could choose a 256-color custom (win98 icon) palette, see, how image will look and continue editing, tuning the gradients, edges and corners of 3d-isometric figures for new icons.

**Comment by ikkqqstudio** [comment]
Isn't Photoshop not available In Linux?

**Reply by nestoris** [reply]
Old Photoshop is available via Wine. The developers of Krita already made the "palettify" filter (thanks to them) which uses Microangelo and Adobe palettes for 256-color viewing of true-color images.
